charts/rancher-vsphere-csi/values.yaml

```diff
@@ -25,7 +25,7 @@ csiController:
     enabled: false
   image:
     repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-    tag: v2.6.2
+    tag: v3.0.1
```

Comment:
I'd suggest not bumping tags in the "default" section for two reasons: 
1. they are always overwritten by the values from the versionOverrides section
2. afaik we did not bump the tags here each time when we bumped the same tags in the versionOverrides section, so there must be some mismatches between the components' tags, we never validate if it will work. 

@a-blender, please correct me if I say something wrong.